spec/decorator_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_relative '../decorator'
+
+class FakeNameable
+  def correct_name
+    'Original Name'
+  end
+end
+
+describe Decorator do
+  let(:fake_nameable) { FakeNameable.new }
+  let(:decorator) { Decorator.new(fake_nameable) }
+
+  describe '#correct_name' do
+    it 'returns the correct name from the decorated object' do
+      expect(decorator.correct_name).to eq('Original Name')
+    end
+  end
+end
```

Comment:
- [ ] Great work on implementing the changes so far 👍 but I guess you missed one thing 😨 The Previous reviewer suggested you make tests for all the classes that you have made while following the Decorator's pattern so I see there is one class which spec is missing and that class in Nameable class 🤔 Your question might be i already have a Nameable class here which i am testing but the thing you are just using Nameable class and testing the Decorator class which is great but you also need to test the Nameable class independently 👍 to follow the project requirements 😊